src/cascadia/TerminalSettingsModel/VisualStudioGenerator.h

```diff
@@ -28,6 +28,8 @@ namespace winrt::Microsoft::Terminal::Settings::Model
     public:
         static std::wstring_view Namespace;
         std::wstring_view GetNamespace() const noexcept override;
+        std::wstring_view GetDisplayName() const noexcept override;
+        std::wstring_view GetIcon() const noexcept override { return {}; };

```

Comment:
The only icons I found here were one for PowerShell and one for CMD. Can we get a Visual Studio asset for here?